controllers/routes.js

```diff
@@ -286,13 +286,48 @@ const routes = {
     }
 };
 
-// for the sake of brevity, define some commonly-used paths rather than repeating them below
+/**
+ * Programme Migration
+ *
+ * Handle redirects from /global-content/programmes to /funding/programmes
+ * @TODO: Consider merging into a global-content/programmes/* handler
+ *        once we decide to migrate all programme pages.
+ */
+function programmeMigration(from, to, isLive) {
+    return {
+        path: `/global-content/programmes/${from}`,
+        destination: `/funding/programmes/${to}`,
+        isPostable: false,
+        allowQueryStrings: false,
+        live: !isLive ? false : true
+    };
+}
+const programmeRedirects = [
+    programmeMigration('england/reaching-communities-england', 'reaching-communities-england', false),
+    programmeMigration('england/parks-for-people', 'parks-for-people', false),
+    programmeMigration('northern-ireland/people-and-communities', 'people-and-communities', false),
+    programmeMigration('wales/people-and-places-medium-grants', 'people-and-places-medium', false),
```

Comment:
Maybe change `people-and-places-medium` to `people-and-places-medium-grants` to match the format of the `large` one?